scripts/generate-tags

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+set -u
+
+# If an optional suffix is provided, append it to all the tags as `-suffix`.
+# Note that this breaks the semver format, because appending -foobar means it's
+# a pre-release version, not a "flavor" as some people claim. The more
+# appropriate way to express this would be to use `+suffix`, but we'll defer
+# that until we have fixed the different places where this might be used.
+suffix=${1:+-}${1:-}
+
+rootdir=$(git rev-parse --show-toplevel)
+version=$("${rootdir}/scripts/version")
+
+version_base=$(echo "${version}" | cut -d- -f1)
+version_seq=$(echo "${version}" | cut -d- -f2)
```

Comment:
By doing this, we are dropping the short sha out of the output of `/scripts/version`, is that okay?

```
10:01:04 ~/Devel/synthetic-monitoring-agent $> git describe --tags --always --long
v0.25.2-28-g3e038e9

10:01:47 ~/Devel/synthetic-monitoring-agent $> git describe --tags --always --long | cut -d- -f1
v0.25.2

10:01:08 ~/Devel/synthetic-monitoring-agent $> git describe --tags --always --long | cut -d- -f2
28
```

Reply:
I'm not sure I follow...

For a non-tagged commit, this outputs:

```
latest,v0.25.2-34-g3fb991a7
```

For a tagged commit:

```
latest,v0.25.2-0-g5a039921,v0.25.2
```

Reply:
My bad, I think I misread this and thought we were using `version_base` later on instead of the full `version`.